docs/documentation/zh/declaration-files/By Example.md

```diff
@@ -206,23 +203,23 @@ declare class Greeter {
 }
 ```
 
-## Global Variables
+## 全局变量
 
-_Documentation_
+_文档_
 
-> The global variable `foo` contains the number of widgets present.
+> 全局变量`foo`包含当前存在的小部件数量。
 
-_Code_
+_代码_
 
 ```ts
 console.log("Half the number of widgets is " + foo / 2);
 ```
 
-_Declaration_
+_声明_
 
-Use `declare var` to declare variables.
-If the variable is read-only, you can use `declare const`.
-You can also use `declare let` if the variable is block-scoped.
+使用`declare var`来声明变量。
+如果变量是只读的，你可以使用`declare const`。
+如果变量是块作用域的，你也可以使用`declare let`。
```

Comment:
```suggestion
使用 `declare var` 来声明变量。如果变量是只读的，你可以使用 `declare const`。如果变量是块作用域的，你也可以使用 `declare let`。
```

Reply:
Continuous line in original English content should be one line in Chinese translation